src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -668,9 +668,9 @@ export default class QuestionnaireForm extends Component {
         system = "RxNorm"
       }
 
-      if (system.length > 0) {
-        displayText = displayText + ' - ' + system + ' - ' + v.code
-      }
+      // if (system.length > 0) {
```

Comment:
If this isn't needed, should it just be removed?
Or is it better to leave it in to help with debugging in the future?

Reply:
It might be useful for debugging or as a way to display the codes in some contexts, I just thought it looked ugly formatted like this in the table.